src/include/elektra.h

```diff
@@ -253,7 +253,7 @@ void elektraFatalError (Elektra * elektra, ElektraError * fatalError);
  *
  **************************************/
 
-const char * elektraGetValue (Elektra * elektra, const char * name);
+const char * elektraGetRawString (Elektra * elektra, const char * name);
```

Comment:
@markus2330 I didn't remove this function, because `elektraSetRawString` (previously `elektraSetValue`) is actually needed for code-generation and having a public setter without the corresponding getter seemed weird. I think `RawString` describes better what the function actually does.

Also I thought it might be useful to have this function, if a user wants to extend the API with a custom type (e.g. something like BigInteger from Java). Users could write their own functions that check the type with `elektraGetType` and the use `elektraGetRawString` followed by some custom conversion.